.github/workflows/kserve_kind_test.yaml

```diff
@@ -9,29 +9,42 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
 
-    - name: Install KinD
-      run: ./tests/gh-actions/install_kind.sh
+      - name: Install KinD
+        run: ./tests/gh-actions/install_kind.sh
 
-    - name: Create KinD Cluster
-      run: kind create cluster --config tests/gh-actions/kind-cluster.yaml
+      - name: Create KinD Cluster
+        run: kind create cluster --config tests/gh-actions/kind-cluster.yaml
 
-    - name: Install kustomize
-      run: ./tests/gh-actions/install_kustomize.sh
+      - name: Install kustomize
+        run: ./tests/gh-actions/install_kustomize.sh
 
-    - name: Install Istio
-      run: ./tests/gh-actions/install_istio.sh
+      - name: Install Istio
+        run: ./tests/gh-actions/install_istio.sh
 
-    - name: Install cert-manager
-      run: ./tests/gh-actions/install_cert_manager.sh
+      - name: Install cert-manager
+        run: ./tests/gh-actions/install_cert_manager.sh
 
-    - name: Build & Apply manifests
-      run: ./tests/gh-actions/install_kserve.sh
+      - name: Install knative
+        run: ./tests/gh-actions/install_knative.sh
```

Comment:
by switching the order of installing kserve and knative, knative installation fails due to requiring the kubeflow namespace, unable to create kubeflow-gateway